src/components/all-content-modal/AllContentModal.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react'
+
+import { Box, Typography } from '@mui/material'
+
+import { styles } from '~/components/all-content-modal/AllContentModal.styles'
+
+interface AllContentModalProps {
+  Icon?: React.ComponentType
+  title: string
+}
+
+const AllContentModal: FC<AllContentModalProps> = ({
+  Icon,
+  title,
+  children
+}) => {
+  return (
+    <>
+      <Box sx={styles.textWithIconWrapper}>
+        {Icon && <Icon />}
```

Comment:
You can leave here only `{icon}`. And pass to this component props as 
`icon={<SomeIcon/>}`

Reply:
> You can leave here only `{icon}`. And pass to this component props as `icon={<SomeIcon/>}`

thank you for your advice. Fixed it 👍